openapi3/schema_merge_test.go

```diff
@@ -0,0 +1,1033 @@
+package openapi3
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// if ExclusiveMax is true on the minimum Max value, then ExclusiveMax is true in the merged schema.
+func TestMerge_ExclusiveMaxIsTrue(t *testing.T) {
+	merged, err := Merge(Schema{
+		AllOf: SchemaRefs{
+			&SchemaRef{
+				Value: &Schema{
+					Type:         "object",
+					ExclusiveMax: true,
+					Max:          Float64Ptr(1),
+				},
+			},
+			&SchemaRef{
+				Value: &Schema{
+					Type:         "object",
+					ExclusiveMin: false,
```

Comment:
Note this is ExclusiveMin.
Perhaps you meant ExclusiveMax?

Reply:
fixed, thanks.